sprocs/index.js

```diff
@@ -74,6 +74,9 @@ module.exports = {
             'config_select.sql',
             'users_select_or_insert.sql',
             'dump_to_csv.sql',
+            'calculate_assessment_question_stats_for_assessment.psql',
```

Comment:
Changed to .sql. I thought we might get better syntax highlighting if we changed them to .psql, but I don't think that's the case.